src/SingleCellAnnotator.imjoy.html

```diff
@@ -723,8 +733,9 @@
       if(this.current_image_data.location_changed){
         await this.uploadAnnotation();
       }
+      this.loading = true
       this.current_image_index = this.cached_image_data.indexOf(this.current_image_data)
-      if(this.current_image_index<0){
+      if(this.current_image_index<=0){
```

Comment:
this is a fix for the back button. previously, the current_image_index can be minus, since when it is 0, it can go to this.current_image_index--, making the back button clickable but weird